python/cudf/setup.py

```diff
@@ -11,7 +11,15 @@
 
 install_requires = ["numba", "cython"]
 
-cython_files = ["cudf/_lib/**/*.pyx"]
+cython_files = ["cudf/**/*.pyx"]
+
+CUDA_HOME = os.environ.get("CUDA_HOME", False)
+if not CUDA_HOME:
+    CUDA_HOME = (
+        os.popen('echo "$(dirname $(dirname $(which nvcc)))"').read().strip()
+    )
+cuda_include_dir = os.path.join(CUDA_HOME, "include")
```

Comment:
What do we need from CUDA? We should change this from `which nvcc` to `which cuda-gdb` as there's going to be a conda `nvcc` shim package in the future that will make this wrong.

Reply:
Also would use [`shutil.which`]( https://docs.python.org/3.6/library/shutil.html#shutil.which ) instead. Can then call [`os.path.dirname`]( https://docs.python.org/3.6/library/os.path.html#os.path.dirname ) on the results as needed.

Reply:
Didn't know about `shutil.which`, thanks!

Reply:
Sweet! Fixed to `cuda-gdb` and also used `shutil.which`